test/membrane_flv_plugin/demuxer_test.exs

```diff
@@ -37,10 +37,7 @@ defmodule Membrane.FLV.Demuxer.Test do
              }, "/tmp/audio.aac"}
 
           :H264 ->
-            {%Membrane.H264.FFmpeg.Parser{
-               alignment: :au,
-               skip_until_parameters?: false
```

Comment:
In theory the new parser always skips until parameters, but since it works with the stream provided, we can leave it ;)